### ISSUE TITLE
Unable to install PM2@1.0.2

### ISSUE THREAD

**Issue by jiripospisil** [pnpm/pnpm#716]
### pnpm version:
0.65.1

### Code to reproduce the issue:
```
npm init -y && pnpm i pm2@1.0.2
```
(As a side note, pnpm seems to ignore the `-y` flag to `init`.)

### Expected behavior:

It should install the version without any issues.

### Actual behavior:

```
ERROR Cannot read property 'type' of undefined
```
[pnpm-debug.log](https://gist.github.com/jiripospisil/20602dff728e8873494b155f3c047ae0)

### Additional information:

 - `node -v` prints: v7.9.0
 - Windows, OS X, or Linux?: OS X

**Comment by zkochan** [comment]
This was a regression. I added a test for it and fixed in [v0.65.2](https://github.com/pnpm/pnpm/releases/tag/v0.65.2)

**Comment by jiripospisil** [comment]
Thanks!
